applications/nrf_desktop/src/modules/Kconfig.hotfixes

```diff
@@ -8,12 +8,22 @@ menu "Hotfix set"
 
 config DESKTOP_HFCLK_LOCK_ENABLE
 	bool "Keep HF clock enabled"
+	depends on !SOC_SERIES_NRF54HX
```

Comment:
shouldn't we also add the following dependency?

`depends on !SOC_SERIES_NRF53X`

Reply:
The nRF53 SoC Series does not support LLPM, so the hotfix might not be that relevant for it - for BLE the shortest connection interval is 7.5 ms (not 100% sure if the hotfix would work properly on nRF53 too).

Maybe we could add a dependency on LLPM here (LLPM reduces the HID data latency far more significantly; the HF clock lock is just a minor follow-up improvement comparing to switching from BLE to LLPM connection parameters)?

Reply:
discussed offline